### PR TITLE
Day 14 Python

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

File: 2020/day-14/AngelOnFira/day14.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+for i, value in enumerate(data):
+    if value.startswith("mask ="):
+        mask_str = value.split(" = ")
+        mask = mask_str[1]
+
+    else:
+        groups = re.search(r'mem\[(\d+)\] = (\d+)', value)
+        location = int(groups.group(1))
+        value = int(groups.group(2))
```
Comment (arora-aditya) [inline 2020/day-14/AngelOnFira/day14.py:20]: classy

File: 2020/day-14/AngelOnFira/day14.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        this_loc = ""
+
+        binary = '{0:036b}'.format(location)
```
Comment (arora-aditya) [inline 2020/day-14/AngelOnFira/day14.py:24]: I lost it here in my solution since I didn't pad the 36 properly, found it quickly tho so no harm done

Comment (AngelOnFira) [reply 2020/day-14/AngelOnFira/day14.py:24]: This is something that would have gotten me running in loops for a while I think. Luckily the first stack overflow I found had this template :P

File: 2020/day-14/AngelOnFira/day14.py (hunk 1 above)
Comment (AngelOnFira) [reply 2020/day-14/AngelOnFira/day14.py:20]: lmao ikr, got a regex sample in my template which comes in handy

**Pull request merged**